open_academy/views/res_parner_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The filename is almost fine, but the t is missing from res_partner. Please rename it and all its references.